music/admin.py

```diff
@@ -17,4 +27,18 @@ def add_music(self, request, queryset):
             music.room.add_music(**MusicSerializer(music).data)
         return
 
+    def duplicate_music(self,request,queryset):
+        targetRoom = Room.objects.get(name=request.POST['nrroom'])
+        if targetRoom:
+            for music in queryset:
+                if targetRoom.name != music.room.name:
+                    new_entry = music
+                    new_entry.room=targetRoom
+                    new_entry.id = None
```

Comment:
Probably cause to this. As there is no id, it can't find the object to save so creates a new one.
It looks like more a hack that an explicit copy :/